pixi.toml

```diff
@@ -99,7 +99,7 @@ test-ribasim-core = { cmd = "julia --project=core --eval 'using Pkg; Pkg.test()'
 test-ribasim-core-cov = { cmd = "julia --project=core --eval 'using Pkg; Pkg.test(coverage=true, julia_args=[\"--check-bounds=yes\"])'", depends_on = [
     "generate-testmodels",
 ] }
-generate-testmodels = "python utils/generate-testmodels.py"
+generate-testmodels = { cmd = "python utils/generate-testmodels.py", inputs = ["python/ribasim","python/ribasim_testmodels"], outputs = ["generated_testmodels"]}
```

Comment:
This works as expected for me.
I think you are good go with the other tasks :)
- build-julia-docs
- quartodoc-build
- codegen